### PR TITLE
Update PaginationContainer.md

### PR DESCRIPTION
**Pull request by nodkz**
Fix code typos

### CODE REVIEW

File: docs/modern/PaginationContainer.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ class Feed extends React.Component {
           edge => <Story story={edge.node} key={edge.node.id} />
         )}
         <button
-          onPress={this._loadMore}
+          onPress={() => this._loadMore()}
```
Comment (nodkz) [inline docs/modern/PaginationContainer.md]: Or maybe `.bind(this)` depends on your code style.

Comment (sibelius) [inline docs/modern/PaginationContainer.md]: this._loadMore works fine

Comment (nodkz) [inline docs/modern/PaginationContainer.md]: Should not. 
Cause `this._loadMore` does not autobind `Feed` instance context for getting its props.

Comment (kassens) [inline docs/modern/PaginationContainer.md]: You could use property initializer syntax, but that's an early feature. This fix seems good here.

**Pull request closed without merging**